python/cudf/cudf/api/extensions/__init__.py

```diff
@@ -1,12 +1,14 @@
-# Copyright (c) 2020, NVIDIA CORPORATION.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
 
 from cudf.api.extensions.accessor import (
     register_dataframe_accessor,
     register_index_accessor,
     register_series_accessor,
 )
+from pandas.api.extensions import no_default
```

Comment:
Note: We have `no_default` introduced in `pandas_2.0_feature_branch`: https://github.com/rapidsai/cudf/pull/13152, but not in `branch-23.06`.

Reply:
Is it possible to cherry-pick that change over rather than doing it manually here? Would make your life easier when the final merge comes around, I suspect the current approach will have some (small) conflicts.

Reply:
Since Prem is out for a while I'll take this over tomorrow and apply the commit(s).

Reply:
We merged the `no_default` change to `pandas_2.0_feature_branch` as a merge commit: https://github.com/rapidsai/cudf/commit/81565cfa490348892b7e883faf78a7000fa2ada0, so cherry-picking it will bring in additional `reset_index` changes that aren't intended for this branch yet. So the best way is to manually resolve the conflict in `pandas_2.0_feature_branch` after this is merged into `branch-23.06`, which anyway has to be done on a daily basis to be up-to-date with `branch-23.06`.

Reply:
OK that's fine with me, was just hoping to save some extra work later.